distribution/lib/Standard/Base/0.0.0-dev/src/Data/Time/Period.enso

```diff
@@ -128,3 +128,17 @@ type Period
                 case err of
                     DateTimeException -> Error.throw Time_Error.Error "Period subtraction failed"
                     ArithmeticException -> Error.throw Illegal_Argument.Error "Arithmetic error"
+
+    ## PRIVATE
+       Convert Period to a friendly string.
+    to_display_text : Text
+    to_display_text self =
+        years = self.years + (self.months/12).floor
+        y = if years = 0 then "" else years.to_text + "Y"
```

Comment:
wouldn't `year/years` be more friendly than `Y`?

Reply:
I think the denser format is better (it was "PT12M").

Reply:
ok